frontend/src/pages/wifiPage/advancedConfigurationDialog/AdvancedConfigDialog.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+
+import Dialog from "../../../components/atoms/dialog/Dialog";
+import PrimaryButton from "../../../components/primaryButton/PrimaryButton";
+import Spinner from "../../../components/atoms/spinner/Spinner";
+
+import styles from "./AdvancedConfigDialog.module.css";
+
+export type Props = {
+  active: boolean;
+  url: string;
+  onClose: () => void;
+  error: boolean;
+};
+
+export enum ErrorMessage {
+  AdvancedConfigError = "There was a problem opening the advanced network configuration. Please, close this dialog and try again later.",
+}
+
+export default ({ active, url, onClose, error }: Props) => {
+  const errorMessage = error && ErrorMessage.AdvancedConfigError;
+
+  return (
+    <Dialog
+      testId="advanced-config-dialog"
+      active={active}
+      title="Advanced Wifi Configuration"
+      className={styles.dialog}
+    >
+      <div className={styles.content}>
+          { !errorMessage && url === "" &&
+              <div className={styles.spinner}>
+                <Spinner size={80} />
+              </div>
+          }
+          { !errorMessage && url !== "" &&
+              <div className={styles.frameContainer}>
+                <iframe
+                  src={url}
+                  data-testid="advanced-config-dialog-frame"
+                  title="Advanced Wifi Configuration"
+                  className={styles.frame}
```

Comment:
The windows keep opening with a horizontal split for me, which isn't as useful a layout

![image](https://user-images.githubusercontent.com/7795213/180990947-5c089b04-19f3-4cf4-8791-0140f88f36ac.png)

Reply:
this might be related to a custom configuration of the tiling manager; can you check if you have any config files in your system?